examples/advanced/k8s_addons/helm_values/nginx_default_values.yaml

```diff
@@ -2,7 +2,7 @@ controller:
   image:
```

Comment:
these are the default values in the chart yea? If so we can remove.

Reply:
Some of the values were not default. I removed any values that were set as default in the helm chart and kept any that aren't default. I opted to keep the image tag set to a specific version in case a image update in the future introduces breaking changes.